main.cpp

```diff
@@ -10,6 +11,8 @@
 
 int main(int argc, char* argv[])
 {
+  std::ofstream ofile("request_response_log.txt");
+  ofile.close();
```

Comment:
I just did that because later in the code, the file is opened to write to. I close it right away because that line is basically only for creating the file, and if something breaks along the way, I don't want the file to be left open.